### PR TITLE
Skip checking relay chain block numbers on testnet

### PR DESCRIPTION
**Pull request by zjb0807**
Closes: #2342

### CODE REVIEW

File: runtime/common/src/lib.rs [hunk 1]
```diff
@@ -133,6 +138,28 @@ impl Convert<Weight, u64> for WeightToGas {
 	}
 }
 
+pub struct CheckRelayNumber<EvmChainID, RelayNumberStrictlyIncreases>(EvmChainID, RelayNumberStrictlyIncreases);
+impl<EvmChainID: Get<u64>, RelayNumberStrictlyIncreases: CheckAssociatedRelayNumber> CheckAssociatedRelayNumber
+	for CheckRelayNumber<EvmChainID, RelayNumberStrictlyIncreases>
+{
+	fn check_associated_relay_number(current: RelayChainBlockNumber, previous: RelayChainBlockNumber) {
+		if EvmChainID::get() == CHAIN_ID_MANDALA
```
Comment (xlc) [inline runtime/common/src/lib.rs:146]: make this a match case

**Review by xlc — COMMENTED** [review]
some test to confirm will be nice

**Review by xlc — APPROVED** [review]

File: runtime/integration-tests/src/runtime.rs [hunk 2]
```diff
@@ -341,6 +341,29 @@ fn parachain_subaccounts_are_unique() {
...
+		set_relaychain_block_number(10);
+		assert_eq!(ParachainSystem::validation_data().unwrap().relay_parent_number, 10);
+
+		// testnet skip checking relay chain block number
+		set_relaychain_block_number(9);
+		assert_eq!(ParachainSystem::validation_data().unwrap().relay_parent_number, 9);
+
+		// set chain_id mainnet
+		#[cfg(feature = "with-mandala-runtime")]
+		module_evm::ChainId::<Runtime>::set(CHAIN_ID_KARURA_MAINNET);
```
Comment (shaunxw) [inline runtime/integration-tests/src/runtime.rs:358]: it should be mandala chain ID.

Comment (zjb0807) [reply runtime/integration-tests/src/runtime.rs:358]: only karura-mainnet and acala-mainnet check relay chain block number, so use karura mainnet chainid with mandala runtime

Comment (shaunxw) [reply runtime/integration-tests/src/runtime.rs:358]: can you add this as comment in the code?

**Review by shaunxw — APPROVED** [review]

**Pull request merged**